### PR TITLE
Update ROADMAP-TO-1.0.0.md

### PR DESCRIPTION
**Pull request by diasdavid**

### CODE REVIEW

File: ROADMAP-TO-1.0.0.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ These are some use case milestones to drive development.
   - [x] js-ipfs-api client inside a tab
   - [x] js-ipfs full node inside a tab
   - [x] browser extension gateway redirects
-  - [ ] browser extension full node
+  - [x] browser extension full node
```
Comment (diasdavid) [inline ROADMAP-TO-1.0.0.md]: 👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽 Web Browsers WG!

File: ROADMAP-TO-1.0.0.md [hunk 2]
```diff
@@ -117,7 +117,7 @@ These are some libraries that must exist to make developers' lives easy. It is c
   - [ ] ipfs api bindings in Erlang
 - [x] full ipfs node implementations
   - [x] go-ipfs - implementation in Go
-  - [ ] js-ipfs - implementation in JS
+  - [x] js-ipfs - implementation in JS
```
Comment (diasdavid) [inline ROADMAP-TO-1.0.0.md]: Oh yeaaah JS Team!! \o/  🚀

File: ROADMAP-TO-1.0.0.md [hunk 3]
```diff
@@ -438,7 +438,7 @@ The IPFS pub/sub service will provide fast delivery and routing for subnets of i
 - [ ] IPFS pub/sub: Protocol and architecture design
   - [ ] IPFS pub/sub: protocol spec / messages
   - [ ] IPFS pub/sub: record spec
-  - [ ] IPFS pub/sub: one-hop support
+  - [x] IPFS pub/sub: one-hop support
```
Comment (diasdavid) [inline ROADMAP-TO-1.0.0.md]: We got the PubSubs and it gets used in multiple apps! (still needs that multihop tree forming stuffs)

File: ROADMAP-TO-1.0.0.md [hunk 4]
```diff
@@ -519,14 +520,14 @@ IPLD data structures and importers are various data structures and programs that
     - [x] unixfs file sharding
     - [x] unixfs file trickledag
     - [x] unixfs dir
-    - [ ] unixfs dir sharding
+    - [x] unixfs dir sharding
```
Comment (diasdavid) [inline ROADMAP-TO-1.0.0.md]: This has been there for a while, we need to tell people about it! ✨

File: ROADMAP-TO-1.0.0.md [hunk 5]
```diff
@@ -519,14 +520,14 @@ IPLD data structures and importers are various data structures and programs that
...
     - [x] unixfs metadata
   - [x] unixfs impl in JS
     - [x] unixfs file
     - [x] unixfs file sharding
-    - [ ] unixfs file trickledag
+    - [x] unixfs file trickledag
     - [x] unixfs dir
-    - [ ] unixfs dir sharding
+    - [x] unixfs dir sharding
```
Comment (diasdavid) [inline ROADMAP-TO-1.0.0.md]: Same in JS! :)

File: ROADMAP-TO-1.0.0.md [hunk 6]
```diff
@@ -674,16 +675,16 @@ The go-ipfs effort is the Go implementation of IPFS. It is meant to run as a com
...
 
-- [ ] js-ipfs ipfs node components
+- [x] js-ipfs ipfs node components
   - [x] js-ipfs multi protocols
   - [x] js-ipfs component: repo
   - [x] js-ipfs component: networking (libp2p)
   - [x] js-ipfs component: bitswap
   - [x] js-ipfs component: dag / ipld
-  - [ ] js-ipfs component: pinning
+  - [x] js-ipfs component: pinning
```
Comment (diasdavid) [inline ROADMAP-TO-1.0.0.md]: Awesome latest addition!

File: ROADMAP-TO-1.0.0.md [hunk 7]
```diff
@@ -796,10 +796,10 @@ We have an ipfs node "management console" GUI, delivered as a distributed webapp
     - [x] adding files (+ drag and drop)
   - [x] tail logs (json)
   - [x] view & edit config (json)
-- [ ] webui console v1
+- [x] webui console v1
   - [x] webapp redesign
   - [x] peer map (flat and 3D)
-  - [ ] new dag explorer
+  - [x] new dag explorer
```
Comment (diasdavid) [inline ROADMAP-TO-1.0.0.md]: This one is coming up super soon to a WebUI close to you!

**Comment by diasdavid** [comment]
Congratz to the whole @ipfs/javascript-team for putting so many things on a boat! 🚢

👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽👏🏽

File: ROADMAP-TO-1.0.0.md (hunk 6 above)
Comment (lidel) [reply ROADMAP-TO-1.0.0.md]: wow, when `ipfs.pin.add` in js-ipfs will be released?

Comment (diasdavid) [reply ROADMAP-TO-1.0.0.md]: 0.30 🌟 https://github.com/ipfs/js-ipfs/issues/1375

**Comment by mgoelzer** [comment]
Congrats on the `[x]`'s!  🍾

Would it make sense to rename this file to `REQUIREMENTS-FOR-1.0.0.md`?  (Could be a separate PR later.)

**Comment by diasdavid** [comment]
Alright, merging this one!

> Would it make sense to rename this file to REQUIREMENTS-FOR-1.0.0.md? (Could be a separate PR later.)

Agree on separate PR :)

**Pull request merged**